common/changes/office-ui-fabric-react/tabrumle-AddLabelToOverflowButton_2018-11-26-20-58.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add ability to change persona coins to show initials onHover when it uses an image",
```

Comment:
Could you please prefix this with `Persona:`? This helps a lot with readability of the release notes. Thanks!